File: Data Structures & Algorithms/two-integer-sum/submission-0.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
        int n = nums.size();
        vector<int>ans(2,0);
        for (int i=0;i<n;i++){
            for(int j=i+1;j<n;j++){
                if(nums[i]+nums[j]==target){
                    ans[0]=i;
                    ans[1]=j;
                    if(j<i){
                        swap(ans[0],ans[1]);
                    }
                    return ans;
                }
            }
        }
        return ans;
    }
};

File: Data Structures & Algorithms/remove-element/submission-0.py
class Solution:
    def removeElement(self, nums: List[int], val: int) -> int:
        k=0
        for i in range(0,len(nums)):
            if nums[i]!=val:
                k=k+1 
                if k-1!=i:
                    nums[k-1]=nums[i]
        return k

        